plugins/theme-highcontrast/test/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable linebreak-style */
```

Comment:
Please re-enable this lint rule, and fix any issues with linebreaks.

Lint rules keep our code formatting consistent, which means that later commits don't contain distracting whitespace changes as different programmers work on them.

You may be able to just delete this line; otherwise you may need to change settings in your editor.